apps/public-ui/pages/sitemap.xml.tsx

```diff
@@ -0,0 +1,46 @@
+const Sitemap = () => {
+  return null;
+};
+
+export const getServerSideProps = async ({ res }) => {
```

Comment:
Should this use getStaticProps? I don't think we need the sitemap generated per request, just per build.

Reply:
Nope, because we want to set a request header. Changed back to server side props